src/main/flight/dyn_notch_filter.c

```diff
@@ -375,7 +375,8 @@ static FAST_CODE_NOINLINE void dynNotchProcess(void)
             }
 
             if (state.axis == gyro.gyroDebugAxis) {
-                for (int p = 0; p < dynNotch.count && p < 3; p++) {
+                for (int p = 1; p <= dynNotch.count && p <= DYN_NOTCH_COUNT_MAX; p++) {
```

Comment:
As DYN_NOTCH_COUNT_MAX was 5 before - we did not use debug slot 4 and 5 before?

This should not affect performance unless we go beyond slot 3

Reply:
we used channel 0-2 for the first 3 dynamic notch positions and channel 4 for pre DN gyro

Reply:
Just think we have a one-off issue with centerFreq[0] as @ledvinap confirms.

Reply:
in earlier versions of BF, the pre DN gyro was placed in the 4th debug column (there was only 4 debug columns before BF4.5). Jan suggested it was best to put the pre DN gyro in 1st column instead, and have the DN notch centers occupy columns 2-8 (I'm speaking in terms of matrix columns to avoid confusion about array/channel numbers which are 1 less).